.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Introduce data to the SGID
+about: Creation Checklist
+title: Add <dataset name> from <place>
+labels: 'introduction'
+assignees: ''
+---
+
+<!-- 
+Introduce your data, where it comes from, why it is being added etc
+-->
+# Summary
+
+We received yada yada from blahbidy blah. Foo Bar is the data contact and it will be updated quarterly. I propose we call it Baz and put it in this ISO category or group.
```

Comment:
thought: Is it worth the hassle to formalize a format for group, contact, and update? Or is that more info than is needed for this project? 

```
Source Agency/Entity: Foo
Contact: Bar Baz
Update Frequency: Quarterly
```

Reply:
We could try it